extensions/mssql/src/objectManagement/commands.ts

```diff
@@ -106,15 +107,19 @@ async function handleObjectPropertiesDialogCommand(context: azdata.ObjectExplore
 		return;
 	}
 	try {
-		const parentUrn = await getParentUrn(context);
+		const parentUrn = context.nodeInfo ? await getParentUrn(context) : undefined;
+		const objectType = context.nodeInfo && context.nodeInfo.nodeType === 'Database' ? ObjectManagement.NodeType.DatabaseProperties : context.nodeInfo.nodeType as ObjectManagement.NodeType;
```

Comment:
This should be unnecessary once you fix the NodeType to be Database

Reply:
Actually these all changes were added for Server properties(inital commit) and the changes were carried over to my PR when i rebase on it. not needed for the database properties and cleaning up!